143 - Add ids splitter (#410)
infor-design/enterprise-wc: src/components/ids-splitter/README.MD

# Ids Splitter Component

> The Ids Splitter is used to divides the user interface of a screen. It is composed of resizable "panes" which divide content laid out either horizontally or vertically

Reviewer: ```suggestion
The Ids Splitter is used to divide the user interface of a page. It is composed of resizable "panes" which divide content. It can layout either horizontally or vertically along an axis where a user can resize the panes, these are represented as `<ids-splitter-pane>` components.
```

> along an axis where a user can resize the panes, these are represented as `<ids-splitter-pane>` components.

Reviewer: ```suggestion
```

![Different variations of IdsSplitterLayout](./doc/ids-splitter-markup-legend.png)

> The pane can move to the left or the right of the screen. This allows the user to have control of the customized layout. Elements inside and outside the pane will be resized to fit.

Reviewer: ```suggestion
The pane can move to the left or the right of the screen. This allows the user to have control of the customized layout. Elements inside and outside the pane will be resized to fit depending on if their css is fluid.
```
